puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-umask-077-systest.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: foreman-umask-077
```

Comment:
Should this end with `nightly-systest` ? @ekohl Is this a test that should be run stand-alone or can it be combined?

Reply:
Not sure what you would combine it with. It can't be combined with regular installs since they have a different umask. We should actually extend it to katello so we know that works. There have been bug reports about it IIRC.

Reply:
Oops, yes. this does need `-systest` on the end.